lutra/Taskfile.yml

```diff
@@ -0,0 +1,57 @@
+version: "3"
+
+vars:
+  packages: -p lutra
+
+tasks:
+  run:
+    cmds:
+      - cmd: cargo run -- print example-project
+
+  fmt:
+    desc: Format prqlc source files
+    cmds:
+      - cmd: |
+          # remove trailing whitespace
+          rg '\s+$' --files-with-matches --glob '!*.{rs,snap}' . \
+          | xargs -I _ sh -c "echo Removing trailing whitespace from _ && sd '[\t ]+$' '' _"
```

Comment:
(FYI if you're in the mood for new tools, I really like `rargs` rather than `xargs` for this sort of thing. I almost never use `xargs` now)

Reply:
...even if I still maintain that `pre-commit` is better for these sorts of lints! 😄